tests/builder-payload.unit.test.ts

```
@@ -79,6 +79,20 @@ describe('builder payload shape', () => {
     ])
   })
 
+
+  it.concurrent('injects the app timeout into buildOptions', () => {
```

ESLint

[error] More than 1 blank line not allowed.

(style/no-multiple-empty-lines)